did_finder_cernopendata/Dockerfile

```diff
@@ -25,19 +25,23 @@ COPY poetry.lock poetry.lock
 
 # Bring over the main scripts
 COPY . .
-# Change ownership of the app directory to celery user
-RUN chown -R celery:celery /usr/src/app
+
+ENV XDG_CONFIG_HOME=/opt/servicex
+RUN poetry config virtualenvs.in-project true
+RUN poetry install --no-root --no-interaction --no-ansi
+
+# Change ownership of the app directory to celery user. Also set the group to zero since
+# that is the group OpenShift will run the container as
+RUN chown -R celery:0 /opt/servicex
+RUN chgrp -R 0 /opt/servicex && chmod -R g=u /opt/servicex
```

Comment:
I think the  `chgrp -R 0` is redundant since the `chown` call above specifies the group

Reply:
Tried removing it along with the `g=u` chmod and it still works! Delete the entire line